vcd/resource_vcd_vm_insert_eject_media.go

```diff
@@ -0,0 +1,141 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/govcd"
+	"log"
+)
+
+func resourceVcdInsertEjectMedia() *schema.Resource {
```

Comment:
Didn't we agree to name the resource (and corresponding functions) `vcd_inserted_media`?

Reply:
This is internal function - code is divided to functions to represent corresponding job it's doing.

Reply:
OK, I actually wanted to comment on the filename, but that's not possible. The filename is `resource_vcd_vm_insert_eject_media.go`, but in the issue tracker we agreed that the resource will be `vcd_inserted_media`.